test/transaction.test.ts

```diff
@@ -0,0 +1,16 @@
+import { getTransactionTemplate } from '../src';
+
+import { SAMPLE_IMAGE_URL } from './stub/samples';
+
+describe('Render new transaction templates', () => {
+  it('Default', () => {
+    const html = getTransactionTemplate({
+      language: 'en',
+      toDisplayName: 'taiwanrenshelly',
+      fromDisplayName: 'taiwannanren',
```

Comment:
Personally, I prefer using `SAMPLE_DISPLAYNAME` for `toDisplayName` and `fromDisplayName`